resources/service.php

```diff
@@ -466,6 +466,16 @@
                     'type' => 'string',
                     'location' => 'query',
                     'required' => false
+                ],
+                    'sort' => [
```

Comment:
Too much indentation here

Reply:
Could you add a testcase for these new parameters?

Reply:
Will do!